named_entity_recognition/documentation/step-4-custom-prediction-routine.md

```diff
@@ -0,0 +1,41 @@
+# Custom prediction routine
+
+Allow us to determine which code runs when sending a prediction request.
+Without custom prediction routine the machine learning framework handles the prediction operation.
+With custom prediction routine you can define custom code which runs for each prediction request.
+
+## Why custom prediction routine
+Our model requires numeric inputs, remember the preprocessing steps.
```

Comment:
How about replacing this paragraph with something like:
Our model requires numeric inputs, which we convert from text before training (this is the preprocessing step). To perform the same conversion at prediction time, inject the preprocessing code by defining a custom prediction routine.

Reply:
✓ done